gradle/project.gradle.kts

```diff
@@ -21,18 +21,13 @@ import kotlin.text.Charsets.UTF_8
 
 // Note: the plugins block is evaluated before the script itself, so no variables can be used
 plugins {
-    id("com.gradle.build-scan") version "2.1"
+    id("com.gradle.build-scan") version "1.16"
```

Comment:
Pls explain the reason for downgrading the plugin

Reply:
https://github.com/sghpjuikit/player/pull/117/commits/445f905727ea207c1e7175464815142db01ffefa

Because the newer version is not supported below Gradle 5. Until IDEA gets its shit together, we might as well work on 4.10, because otherwise I can't really work on the application.